StreamChatClient_v3/Workers/ChannelUpdater.swift

```diff
@@ -26,6 +26,19 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
         }
     }
 
+    /// Updates specific channel with new data.
+    /// - Parameters:
+    ///   - channelPayload: New channel data..
+    ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    func updateChannel(channelPayload: ChannelEditDetailPayload<ExtraData>, completion: ((Error?) -> Void)? = nil) {
+        apiClient.request(endpoint: .updateChannel(channelPayload: channelPayload)) {
+            switch $0 {
+            case .success: completion?(nil)
+            case let .failure(error): completion?(error)
+            }
```

Comment:
```suggestion
completion?($0.error)
```

Reply:
Yep, was waiting for approve of https://github.com/GetStream/stream-chat-swift/pull/383. 
Will merge https://github.com/GetStream/stream-chat-swift/pull/383 and then will make this change.